src/main/java/org/sonarlint/intellij/issue/hotspot/SecurityHotspotCategory.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2020 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.issue.hotspot;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum SecurityHotspotCategory {
```

Comment:
Would be great to get rid of this hardcoded mapping before the final release.

Reply:
As discussed, it would mean getting the long description instead of the short one from SQ, but they don't plan to do it soon, so we have to keep it